test/inlining/rlexe.xml

```diff
@@ -10,28 +10,52 @@
   <test>
     <default>
       <files>linenumber1.js</files>
-      <tags>fail</tags>
       <compile-flags>-force:inline</compile-flags>
+      <baseline>linenumber1.baseline</baseline>
       <tags>exclude_arm</tags>
+    </default>
+  </test>
+  <test>
+    <default>
+      <files>linenumber1.js</files>
+      <!-- Variant of test without -force:inline to ensure output is the same -->
+      <!-- <compile-flags>-force:inline</compile-flags> -->
       <baseline>linenumber1.baseline</baseline>
+      <tags>exclude_arm</tags>
```

Comment:
why exclude arm? would a `fails_arm` be better, or is it expected that there are diffs on arm?